tests/e2e/UseCase/test_API_provision.py

```diff
@@ -0,0 +1,256 @@
+"""
+
+    Provision Services Rest API Use Case Tests
+
+"""
+
+import string
+import random
+
+import pytest
+import json
+import allure
+
+
+@pytest.mark.ow_sanity_lf
+@pytest.mark.uc_sanity
+@pytest.mark.owprov_api_tests
+@pytest.mark.owprov_api_usecase
+@allure.feature("SDK PROV REST API")
+class TestProvAPIUseCase(object):
+
+    @pytest.mark.prov_api_usecase_test
+    def test_prov_service_use_case(self, setup_prov_controller, testbed):
```

Comment:
@haricharan-jaka Please add the relevent allure labelings here

Reply:
Ok @shivamcandela Will do that